doc/source/howtos/upgrade_guide.rst

```diff
@@ -126,54 +118,84 @@ authentication plugin in these cases.
 
 Be sure you are running on the latest 5.7 version before you upgrade to 8.0.
 
-|mysql-upgrade| is part of :term:`SST`
---------------------------------------------------------------------------------
+.. upgrading.auto::
 
-|mysql-upgrade| is now run automatically as part of :term:`SST`. You do not have
-to run it manually when upgrading your system from an older version.
+Auto-upgrade
```

Comment:
Remove this section.
Removed wsrep_state.dat and supporting code from SST process. https://jira.percona.com/browse/PXC-2997

Reply:
Removed section on auto-upgrade.